machine_management/control_plane_machine_management/cpmso-configuration.adoc

```diff
@@ -0,0 +1,59 @@
+:_content-type: ASSEMBLY
+[id="cpmso-configuration"]
+= The Control Plane Machine Set Operator configuration
+include::_attributes/common-attributes.adoc[]
+:context: cpmso-configuration
+
+toc::[]
+
+These example YAML file and snippets demonstrate the base structure for a control plane machine set custom resource (CR) and platform-specific samples for failure domain and provider specification configurations.
+
+//Sample YAML for a control plane machine set custom resource
+include::modules/cpmso-yaml-sample-cr.adoc[leveloffset=+3]
```

Comment:
Is the `+3` intentional here (vs `+1`)? Title seemed small in the preview, and since it's the first `include:` can't tell if it was intended to be a subsection or not.

Reply:
Good catch, I think that's leftover from before I broke this into modules